cobra/test/test_io/test_annotation_format.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from os.path import join
+from cobra.io import load_json_model, write_sbml_model
+
+
+def test_load_json_model_valid(data_directory):
+    """Testing valid annotation format"""
```

Comment:
The first line of a docstring should be in imperative mood.

```suggestion
    """Test loading a valid annotation from JSON."""
```